### DISCUSSION TITLE
Unit calculation (rem to px) between figma and token studios

### DISCUSSION THREAD

**Discussion by ndupasquier** [primefaces/community#3165] [Figma UI Kit]
Hello,

I've purchased the primeng figma kit but i'm facing an issue with the unit calculation.

For example, for the ❖ InputText component (large size) , you can see that the padding show as 12.25 (px) whereas, token studio show a value 0.875rem. 

The padding should be 14px if the base unit is 1rem = 16px. 

Do you have any idea why 0.875rem show as 12.25px on figma ? 
<img width="673" alt="Capture d’écran 2024-12-04 à 17 02 02" src="https://github.com/user-attachments/assets/5cf57ca5-640b-43b9-abc7-8d7da5dce9c7">
<img width="1115" alt="Capture d’écran 2024-12-04 à 17 00 20" src="https://github.com/user-attachments/assets/f2b3c83b-7cea-4847-93ed-17e72fc87a6e">

**Comment by ndupasquier** [comment]
Can someone help on this ? It's not only token studio, the variable in figma show 12.25px, instead of 14 px because 0,875rem = 14 px

**Comment by onursenture** [comment]
> The padding should be 14px if the base unit is 1rem = 16px.

1rem is 14px. You should set the Base font size as `{app.font.size}` to avoid any miscalculation on your token actions. 

<img width="399" alt="image" src="https://github.com/user-attachments/assets/16d34797-9bd1-4b71-a5c9-58acbe34c700" />
